### PR TITLE
added aggregation

### PR DESCRIPTION
**Pull request by StephenCoady**
added a basic way to aggregate mutations based on the id of the object and also the mutation name.

### CODE REVIEW

**Comment by StephenCoady** [comment]
this is not right yet, bit buggy

File: web/src/sdk/mutations/QueueMutationLink.js [hunk 1]
```diff
@@ -57,7 +58,22 @@ export class QueueMutationLink extends ApolloLink {
     //store only if there are values for query.definitions
     if (definitions.length > 0) {
       query.definitions = definitions
-      this.queue.push({mutation: query, variables})
+      let operationName = query.definitions[0].name.value
```
Comment (wtrocki) [inline web/src/sdk/mutations/QueueMutationLink.js]: Not guaranteed to be always there. Let's add some if's to check if things are not undefined.

**Comment by wtrocki** [comment]
Looks really good!

**Comment by wtrocki** [comment]
❤️ Going to verify this now.

**Pull request merged**